src/lib/util/iheap.ml

```diff
@@ -33,8 +33,9 @@ type t = {heap : int Vec.t; indices : int Vec.t }
 let dummy = -100
 
 let init sz =
-  { heap    =  Vec.init sz (fun i -> i) dummy;
-    indices =  Vec.init sz (fun i -> i) dummy}
+  let lst = List.init sz (fun i -> i) in
+  { heap    =  Vec.of_list lst ~dummy;
+    indices =  Vec.of_list lst ~dummy}
```

Comment:
Why this change?

Reply:
It is because the API of `Vec` changes and there is no `init` function anymore. Probably the older code is a bit faster.